packages/utils/src/http.ts

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosError } from 'axios'
+
+export class HttpError {
+  constructor(public msg: string, public httpStatus: number) {}
+}
+
+export async function post(url: string, json_data: string): Promise<string> {
+  try {
+    let response = await axios.post(url, json_data, {
+      timeout: 30_000,
+      maxRedirects: 3,
```

Comment:
We should document our timeouts and retries for the RPC in a central place as this will make reasoning and debugging easier. Also please make this configurable.

Reply:
Added config option that can be later used from Rust when everything is integrated.